external/loaders/loaders/mappers/_hybrid.py

```diff
@@ -87,7 +88,7 @@ class FineResBudget(Protocol):
 
 
 def open_zarr(url, consolidated=False):
-    mapper = fsspec.get_mapper(url)
+    mapper = zarr.LRUStoreCache(fsspec.get_mapper(data_path), 128 * 2 ** 20)
```

Comment:
this cache size (128MB) worked well for prescribed runs with Q1 and Q2 precomputed. If we are computing on-the-fly, we might need to make cache bigger, since we need about 7-8x as much input data. (7-8 terms are needed each to compute Q1 and Q2).

Reply:
Is the idea here that the MPI will call the same chunks multiple times?